bootstrap/updater.go

```diff
@@ -126,7 +128,7 @@ func New(clock layerClock, opts ...Opt) *Updater {
 	return u
 }
 
-func (u *Updater) Subscribe() chan *VerifiedUpdate {
+func (u *Updater) Subscribe() <-chan *VerifiedUpdate {
 	u.mu.Lock()
 	defer u.mu.Unlock()
 	ch := make(chan *VerifiedUpdate, 10)
```

Comment:
check if stop channel is close or not?

Reply:
Updated to return an error if Updater has already been closed